src/js_native_api.h

```diff
@@ -486,6 +486,11 @@ napi_get_dataview_info(napi_env env,
                        napi_value* arraybuffer,
                        size_t* byte_offset);
 
+NAPI_EXTERN napi_status NAPI_CDECL
```

Comment:
There must be the `#ifdef NAPI_EXPERIMENTAL` guards around the new functions.

Reply:
Thanks, I applied